freezer.tex

```diff
@@ -1,125 +1,162 @@
+(and the MEGA65) use ``disk images'', which are files that represent a floppy disk, usually stored on an SD card.
+By ``mounting'' a disk image file to an emulator, the MEGA65 can then use these files without realising it's actually
+being emulated. As far as the MEGA65 is concerned, it's using a real drive.
 
-\section{Terminology}
+This allows you to use modern media to read and write your programs, and in some cases can drastically reduce
+read and write times. These files usually have the extension {\bf .d64} (named after the C64), which are files that
+represent a {\it single} side of a 5.25" disk, or {\bf .d81}, which represents an entire 3.5" disk (named after the
+1581 floppy disk drive).
 
-{\bf BASIC} and {\bf CBDOS} use following terminology:
+The MEGA65 can use either real disks and drives, {\bf .d81} files, or a combination of both.
 
-{\bf UNIT} is a device number in the range 0-31.
-The numbers from 0 to 11 are reserved for following device types:
+\subsection{Disk Drive Terminology}
+Before you get to know how to use the various disk drives that work with the MEGA65, it's important to be aware
+of some specific terminology which CBM computers and the MEGA65 uses. This includes {\bf BASIC} commands (such as
+{\bf LOAD} and {\bf SAVE}), and {\bf CBDOS} (Computer Based Disk Operating System), which use the following:
+
+{\bf UNIT} is a device number in the range of 0-31.
+The numbers from 0 to 11 are reserved for the following device types:
 
-{\ttfamily
 \setlength{\tabcolsep}{1mm}
 \begin{center}
 \begin{tabular}{|l|l|l|}
 \hline
- unit \# & device  & comment \\
+{\bf Unit} \# & {\bf Device}  & {\bf Notes} \\
 \hline
-0        & KEYBOARD & input \\
-1        & unused   & was TAPE on C64 \\
-2        & unused   & was RS232 on C64 \\
-3        & SCREEN   & input/output     \\
-4-5      & IEC PRINTER  & output     \\
-6-7      & IEC PLOTTER  & output     \\
-8-9      & CBDOS drives & floppy drive or disk image \\
+0        & KEYBOARD & Input \\
+1        & Unused   & Was used for Cassettes on the C64 \\
+2        & Unused   & Was used for RS-232 on the C64 \\
+3        & SCREEN   & Input/Output     \\
+4-5      & IEC PRINTER  & Output     \\
+6-7      & IEC PLOTTER  & Output     \\
+8-9      & CBDOS drives\footnotemark{} & Internal floppy drive, 1565\footnotemark{}, or disk image \\
 10-11    & IEC drives   & 1541, 1571, 1581, FD-2000 \\
 \hline
 \end{tabular}
 \end{center}
-}
+\addtocounter{footnote}{-2}
+\stepcounter{footnote}\footnotetext{IEC drives can be assigned to devices 8-9, by moving the built-in MEGA65 devices
+                                    to 10-11 first. Once the drives have been reassigned, the IEC drives can be
+                                    powered on.}
+\stepcounter{footnote}\footnotetext{The 1565 was a prototype external C65 disk drive that never made it to production.
+                                    The internal MEGA65 drive is a reproduction of this prototype.}
```

Comment:
Probably not a big deal for the printed version, but just to share an obs on my end, when I built the pdf of your changes, I saw the two footnote hyperlinks here, but clicking on them didn't seem to go anywhere:
![image](https://user-images.githubusercontent.com/20013309/137429346-505b896e-a2a7-4a05-8bf3-cc64025cb585.png)

Perhaps it was an edge-case due to the table rolling onto the next page, and the footnotes appearing there instead:
![image](https://user-images.githubusercontent.com/20013309/137429422-70f11c2b-5f36-4cab-a2b5-e413e9deb1e3.png)